frontend/src/components/common/Card.tsx

```diff
@@ -1,39 +1,41 @@
 const Card = ({
   title,
-  description,
   image,
   genres,
 }: {
   title: string;
-  description: string;
```

Comment:
Perchè hai tolto la description? Non viene fornita dalle api? In generale se apri una PR fai ciò che è affine al task su cui stai lavorando, evita di fare attività terze

Reply:
> Perchè hai tolto la description? Non viene fornita dalle api?

No @makapx , ho deciso di usare due tipi per gli Anime: un `AnimeFragment` che ha informazioni essenziali, e un `Anime` completo di descrizione etc. Se non ti piace la cosa facci sapere che sistemiamo con andrei.
Comunque bisognerebbe cambiare il tipo che prende in input.


> In generale se apri una PR fai ciò che è affine al task su cui stai lavorando, evita di fare attività terze

Perdonaci, per questa volta puoi chiudere un occhio spero.

Reply:
Ok per il fragment. Fixate le cose principali e vi accetto